client/src/pages/tasks/Show.js

```diff
@@ -120,18 +143,24 @@ const BaseTaskShow = ({ pageDispatchers, currentUser }) => {
     searchProps: DEFAULT_SEARCH_PROPS,
     pageDispatchers
   })
-  if (done) {
-    return result
-  }
```

Comment:
Why not leave this here? The calculations below based on `data` are unnecessary if `done`.